src/System.Drawing.Common/tests/Helpers.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.IO;
+using System.Reflection;
+
+namespace System.Drawing.Tests
+{
+    public static class Helpers
+    {
+        public static string GetEmbeddedResourcePath(string name)
+        {
+            return Path.Combine(Assembly.GetExecutingAssembly().Location, "..", "bitmaps", name);
```

Comment:
Small nit: I would prefer this to be

`Path.Combine(AppContext.BaseDirectory, "bitmaps", name)`

Effectively it's the same, but doesn't have the unusual ".." in the path.